### ISSUE TITLE
Does CoAP support Pair-Resume like BLE?

### ISSUE THREAD

**Issue by Jc2k** [Jc2k/aiohomekit#70]
The BLE version of HAP supports a variant of Pair-Verify where it can resume a previous session. Pair-Resume bypasses ECC calculations so it is significantly faster at setting up a secure session for low power BLE devices.

I've implemented this for BLE in https://github.com/Jc2k/aiohomekit/pull/69.

Given the cross-over between HAP BLE devices and HAP Thread devices I am wondering whether these devices support Pair-Resume over CoAP as well.

CC @roysjosh

**Comment by roysjosh** [comment]
I'll take a look! Since the bulbs start out over BLE it wouldn't surprise me.

**Comment by Jc2k** [comment]
I thought the same! 😄 But seems like BLE sessions can be quite short, whereas from what you've said CoAP sessions seem to be more durable so I wouldn't be surprised if the saving wasn't worth it for thread.

**Comment by roysjosh** [comment]
NL bulbs don't seem to implement this over Thread and don't fall back to pair-verify. Probably not in the spec for Thread if they don't fall back.

**Comment by Jc2k** [comment]
Agreed. Having seen how desperate BLE is to be disconnected, can sort of see why thread doesn't need it. Good to confirm though. Thanks!
